apps/web/src/pages/Users/UserInformationPage/components/ChangeStatusDialog.tsx

```diff
@@ -269,7 +268,13 @@ const ChangeStatusDialog = ({
                 })}
               </p>
               <div data-h2-margin="base(x.5, 0, x.125, 0)">
-                <Combobox
+                <PoolFilterInput
+                  includeIds={userPoolIds}
+                  filterInput={{
+                    statuses: [PoolStatus.Closed, PoolStatus.Published],
+                  }}
+                />
+                {/* <Combobox
```

Comment:
Is this meant to be removed?

Reply:
Oops, yeah I did :sweat_smile: Removed in [remove commented combobox](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10383/commits/0124c4b3d14607d1c7bd0bc146347bc5f4603774)